build.gradle

```diff
@@ -23,7 +23,7 @@ buildscript {
 plugins {
     // Versions 1.6.10 thru 2.0.0 depend on a beta version of slf4j, which we don't resolve.
```

Comment:
This is an interesting comment. If it would not resolve, it wouldn't build, right? The CI looks green.

Reply:
Yeah, weird, eh?

So there seems a quirk here that it took me a while to track down: it turns out that gradle _will_ resolve the beta Slf4j artefact _if it's already in the local mavan/ivy download cache_, but won't fetch it if it's not.

This lead to:
 - Jenkins being unhappy.
 - My local build being okay, until I blew away the maven and ivy caches. Then my local build matched Jenkins.

Travis was passing, but only because it wasn't hitting the QA-related gradle targets that use Spotbugs. I've just updated the Travis to cover these as well and rebased… now they're not so green. ;)